scripts/dev/download_libs.sh

```diff
@@ -9,278 +12,284 @@ BLEEDING_EDGE=0
+	else       
+		PKGS="openFrameworksLibs_${VER}_${PLATFORM}_${ARCH}_1.zip \
+			openFrameworksLibs_${VER}_${PLATFORM}_${ARCH}_2.zip \
+			openFrameworksLibs_${VER}_${PLATFORM}_${ARCH}_3.zip \
+			openFrameworksLibs_${VER}_${PLATFORM}_${ARCH}_4.zip"
+	fi
 elif [[ "$PLATFORM" =~ ^(osx|ios|tvos|xros|catos|watchos)$ ]]; then
-    if [[ $BLEEDING_EDGE = 1 ]] ; then
-        PKGS="openFrameworksLibs_${VER}_${PLATFORM}_1.tar.bz2 \
-              openFrameworksLibs_${VER}_${PLATFORM}_2.tar.bz2 \
-              openFrameworksLibs_${VER}_${PLATFORM}_3.tar.bz2"
-    else    
-        PKGS="openFrameworksLibs_${VER}_${PLATFORM}1.tar.bz2 \
-              openFrameworksLibs_${VER}_${PLATFORM}2.tar.bz2 \
-              openFrameworksLibs_${VER}_${PLATFORM}3.tar.bz2 \
-              openFrameworksLibs_${VER}_${PLATFORM}4.tar.bz2" 
-    fi 
+	if [[ $BLEEDING_EDGE = 1 ]] ; then
+		PKGS="openFrameworksLibs_${VER}_${PLATFORM}_1.tar.bz2 \
+			openFrameworksLibs_${VER}_${PLATFORM}_2.tar.bz2 \
+			openFrameworksLibs_${VER}_${PLATFORM}_3.tar.bz2"
+	else    
+		PKGS="openFrameworksLibs_${VER}_${PLATFORM}1.tar.bz2 \
+			openFrameworksLibs_${VER}_${PLATFORM}2.tar.bz2 \
+			openFrameworksLibs_${VER}_${PLATFORM}3.tar.bz2 \
+			openFrameworksLibs_${VER}_${PLATFORM}4.tar.bz2" 
+	fi 
 elif [ "$ARCH" == "" ] && [ "$PLATFORM" == "android" ]; then
-    if [[ $BLEEDING_EDGE = 1 ]] ; then
-        PKGS="openFrameworksLibs_${VER}_${PLATFORM}_armv7.tar.bz2 \
-          openFrameworksLibs_${VER}_${PLATFORM}_arm64.tar.bz2 \
-           openFrameworksLibs_${VER}_${PLATFORM}_x86_64.tar.bz2
-          openFrameworksLibs_${VER}_${PLATFORM}_x86.tar.bz2"
-    else
-        PKGS="openFrameworksLibs_${VER}_${PLATFORM}armv7.tar.bz2 \
-          openFrameworksLibs_${VER}_${PLATFORM}arm64.tar.bz2 \
-          openFrameworksLibs_${VER}_${PLATFORM}x86.tar.bz2"
-    fi
+	if [[ $BLEEDING_EDGE = 1 ]] ; then
+		PKGS="openFrameworksLibs_${VER}_${PLATFORM}_armv7.tar.bz2 \
+		openFrameworksLibs_${VER}_${PLATFORM}_arm64.tar.bz2 \
+		openFrameworksLibs_${VER}_${PLATFORM}_x86_64.tar.bz2
+		openFrameworksLibs_${VER}_${PLATFORM}_x86.tar.bz2"
+	else
+		PKGS="openFrameworksLibs_${VER}_${PLATFORM}armv7.tar.bz2 \
+		openFrameworksLibs_${VER}_${PLATFORM}arm64.tar.bz2 \
+		openFrameworksLibs_${VER}_${PLATFORM}x86.tar.bz2"
+	fi
 else # Linux
-    if [[ $BLEEDING_EDGE = 1 ]] ; then
-        PKGS="openFrameworksLibs_${VER}_${PLATFORM}${ARCH}.tar.bz2"
-    else
-        PKGS="openFrameworksLibs_${VER}_${PLATFORM}${ARCH}.tar.bz2"
-    fi
+	if [[ $BLEEDING_EDGE = 1 ]] ; then
+		PKGS="openFrameworksLibs_${VER}_${PLATFORM}${ARCH}.tar.bz2"
+	else
+		PKGS="openFrameworksLibs_${VER}_${PLATFORM}${ARCH}.tar.bz2"
+	fi
+fi
+
+
+REPO="nightly"
+if [[ $BLEEDING_EDGE = 1 ]] ; then
+	REPO="bleeding"
 fi
 
+PARAMS=""
 for PKG in $PKGS; do
-    download $PKG
+	PARAMS+="https://github.com/openframeworks/apothecary/releases/download/$REPO/$PKG "
 done
+# echo " PARAMS $PARAMS"
+
+
+if command -v wget2 2>/dev/null; then 
```

Comment:
Should just do this in downloader 
https://github.com/openframeworks/openFrameworks/blob/master/scripts/dev/downloader.sh#L25

Reply:
if wget2, else curl, else wget 
```
if command -v wget2 2>/dev/null; then 
        wget2 $@ 2> /dev/null;
elif command -v curl 2>/dev/null; then 
        curl -LO --retry 20 -O -s $@; 
else 
        wget -q $@ 2> /dev/null; 
fi
```